### PR TITLE
Refactor StringSliceFromState to expandStrings

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scaleway/resource_instance_server.go [hunk 1]
```diff
@@ -454,7 +454,7 @@ func resourceScalewayInstanceServerUpdate(d *schema.ResourceData, m interface{})
 	}
 
 	if d.HasChange("tags") {
-		tags := StringSliceFromState(d.Get("tags").([]interface{}))
+		tags := expandStrings(d.Get("tags").([]interface{}))
```
Comment (kindermoumoute) [inline scaleway/resource_instance_server.go]: no need to cast to `[]interface{}`

File: scaleway/resource_k8s_cluster_beta.go [hunk 2]
```diff
@@ -573,7 +573,7 @@ func resourceScalewayK8SClusterBetaUpdate(d *schema.ResourceData, m interface{})
 	}
 
 	if d.HasChange("tags") {
-		tags := StringSliceFromState(d.Get("tags").([]interface{}))
+		tags := expandStrings(d.Get("tags").([]interface{}))
```
Comment (kindermoumoute) [inline scaleway/resource_k8s_cluster_beta.go]: no need to cast to `[]interface{}`

File: scaleway/resource_lb_backend_beta.go [hunk 3]
```diff
@@ -243,7 +243,7 @@ func resourceScalewayLbBackendBetaCreate(d *schema.ResourceData, m interface{})
 			HTTPConfig:      expandLbHCHTTP(d.Get("health_check_http")),
 			HTTPSConfig:     expandLbHCHTTPS(d.Get("health_check_https")),
 		},
-		ServerIP:           StringSliceFromState(d.Get("server_ips").([]interface{})),
+		ServerIP:           expandStrings(d.Get("server_ips").([]interface{})),
```
Comment (kindermoumoute) [inline scaleway/resource_lb_backend_beta.go]: no need to cast to `[]interface{}`

File: scaleway/resource_lb_backend_beta.go [hunk 4]
```diff
@@ -355,7 +355,7 @@ func resourceScalewayLbBackendBetaUpdate(d *schema.ResourceData, m interface{})
 	_, err = lbAPI.SetBackendServers(&lb.SetBackendServersRequest{
 		Region:    region,
 		BackendID: ID,
-		ServerIP:  StringSliceFromState(d.Get("server_ips").([]interface{})),
+		ServerIP:  expandStrings(d.Get("server_ips").([]interface{})),
```
Comment (kindermoumoute) [inline scaleway/resource_lb_backend_beta.go]: no need to cast to `[]interface{}`

File: scaleway/resource_lb_beta.go [hunk 5]
```diff
@@ -130,7 +130,7 @@ func resourceScalewayLbBetaUpdate(d *schema.ResourceData, m interface{}) error {
 			Region: region,
 			LbID:   ID,
 			Name:   d.Get("name").(string),
-			Tags:   StringSliceFromState(d.Get("tags").([]interface{})),
+			Tags:   expandStrings(d.Get("tags").([]interface{})),
```
Comment (kindermoumoute) [inline scaleway/resource_lb_beta.go]: no need to cast to `[]interface{}`

File: scaleway/resource_rdb_instance_beta.go [hunk 6]
```diff
@@ -227,7 +227,7 @@ func resourceScalewayRdbInstanceBetaUpdate(d *schema.ResourceData, m interface{}
 	}
 
 	if d.HasChange("tags") {
-		req.Tags = scw.StringsPtr(StringSliceFromState(d.Get("tags").([]interface{})))
+		req.Tags = scw.StringsPtr(expandStrings(d.Get("tags").([]interface{})))
```
Comment (kindermoumoute) [inline scaleway/resource_rdb_instance_beta.go]: no need to cast to `[]interface{}`

**Comment by remyleone** [comment]
> i guess `StringSliceFromState` can be deleted

Done

**Pull request merged**